app/data/__tests__/team.test.js

```diff
@@ -0,0 +1,56 @@
+const { Team } = require('../team');
+
+describe('Equipos', () => {
+  describe('Comprobando la declaración de un nuevo equipo', () => {
+    it('Comprobando cuando se le pasa como parámetro un string', () => {
+      const team = new Team('CaracasFC');
+      expect(team).toBeTruthy();
+    });
+
+    it('Comprobando que no lanza un error al no pasar una serie de equipos.', () => {
+      const team = new Team();
+      expect(team).toBeTruthy();
+      expect(team.teamName).toBeUndefined();
+      expect(team.listPlayers).toStrictEqual([]);
+      expect(team.teamValue).toStrictEqual(0);
+    });
+  });
+
+  describe('Comprobando el método getTeamName()', () => {
+    it('Comprobando que devuelva el valor correcto', () => {
+      const team = new Team('CaracasFC');
+      expect(team.getTeamName()).toBeTruthy();
+      expect(team.getTeamName()).toBe('CaracasFC');
+    });
+  });
+
+  describe('Comprobando el método getTeamValue()', () => {
+    it('Comprobando que devuelva el valor correcto', () => {
+      const team = new Team('CaracasFC');
+      expect(team.getTeamValue()).toStrictEqual(0);
```

Comment:
@gabodt16  Cambiaría también para que fuera toBe
```suggestion
      expect(team.getTeamValue()).toBe(0);
```

Reply:
Solucionado en el commit:
https://github.com/imsergiobernal/ec-a3-jest-testing/pull/11/commits/7ed8ab3c0f832cab3cc1b7a7ea82bc743332b3d6